dist/gcc-compatible/Hacl_Hash_Base.h

```diff
@@ -47,6 +47,8 @@ uint32_t Hacl_Hash_Definitions_hash_len(Spec_Hash_Definitions_hash_alg a);
 
 typedef uint8_t *Hacl_Hash_Definitions_hash_t;
 
+typedef Spec_Hash_Definitions_hash_alg Hacl_Hash_Definitions_const_impl;
```

Comment:
Maybe worth marking this definition as inline_for_extraction noextract in Hacl.Hash.Definitions?

Reply:
Fixed in 2ce8329cd6 which is part of the subsequent PR.